lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.ts

```diff
@@ -36,6 +36,12 @@ const templateTypes = {
     defaultTemplate: 'defaultTemplate'
 };
 
+interface KeyboardEvents {
```

Comment:
Why you created interface for that? JS has built in type which you should use: 
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent

Reply:
i have now used KeyboardEvent instead of custom interface